.github/workflows/ccpp.yml

```diff
@@ -2,12 +2,12 @@ name: C/C++ CI
 
 on:
   push:
-    branches: [ master ]
-  pull_request:
```

Comment:
You still want to run CI on pull requests.

Reply:
Wasn't really sure what we wanted. I had this at the beginning but had the impression that doing any bors command would still rerun the job, making this initial job a bit less useful. But I may have overlooked this !

Reply:
The bors command is usually only triggered exactly once and that is right before the PR is merged with `bors r+`. bors doesn't replace your CI. It's only there to automate checking PRs before merging.

Reply:
You almost never use `bors try`. The only scenario where you use `bors try` is when you're `bors` CI runs more checks than your PR CI and you want to check if a WIP PR would already pass the `bors` CI.

Reply:
Ok, so I changed it to follow your advice. Everything is executed when a PR is opened and we have no extra tests for bors. Thanks for the comments!